src/documentation/pages/components/forms/range-slider/examples/_base.vue

```diff
@@ -0,0 +1,55 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
+    .chi-form__item(slot="example")
+      label(class="chi-label", id="range-label", for="range") Range label
+      input(class="chi-range-slider", type="range", ref="range")
```

Comment:
On label you are using `for="range01"`
```suggestion
      input(class="chi-range-slider", type="range", ref="range" id="range01")
```